packages/openneuro-app/src/scripts/datalad/mutations/share.jsx

```diff
@@ -42,7 +42,7 @@ export const mergeNewPermission = (
   }
 }
 
-const ShareDataset = ({ datasetId, userEmail, metadata, done }) => (
+const ShareDatasetPermissions = ({ datasetId, userEmail, metadata, done }) => (
```

Comment:
I think converting fully to ManageDatasetPermissions (and likewise for SHARE_DATASET) would help future readers of the code.

Reply:
Agreed; I made a few more changes that will help clarify this as 'managing permissions' and not 'sharing'. Thanks